scripts/generate_conf_files.sh

```diff
@@ -63,7 +63,17 @@ then
 fi
 if [ -z "$CONTENT_CACHE_SIZE" ]
 then
-	export CONTENT_CACHE_SIZE="10g"
+	# Check available disk space in Gigabytes. Using -l to only check local filesystems.
+	DISK_SPACE_GB=$(df -l --block-size=1G / | awk 'NR==2 {print $2}')
+
+	if [ "$DISK_SPACE_GB" -gt 32 ]; then
+		# If disk space is > 32GB, use half of it for cache
+		CACHE_SIZE_GB=$((DISK_SPACE_GB / 2))
```

Comment:
What do you think about making the % disk space to use an environment variable? That would still let us adjust it for specific use cases or testing if necessary.